style.css

```diff
@@ -0,0 +1,226 @@
+body {
+  margin-left: 100px;
+  margin-right: 100px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 20px;
+  align-items: center;
+}
+
+.newsletter {
+  color: #00bcfc;
+  font-weight: 900;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  text-decoration: none;
+}
+
+.newsletter-para {
+  margin-top: -1px;
+  color: #999;
+  font-family: "PT Serif", Georgia, serif;
+}
+
+.newsletter-para:hover {
+  text-decoration: underline;
+}
+
+h1.newscientist {
+  font-size: 60px;
+  letter-spacing: -5px;
+  font-family: "PT SERIF", Georgia, serif;
+}
+
+.halfpricesale {
+  color: antiquewhite;
+  font-weight: 900;
+  background-color: #00bcfc;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  width: 15rem;
+  text-align: center;
+  height: 20px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.nav-links {
+  display: flex;
+  margin-right: 10px;
+}
+
+li {
+  width: 100%;
+  list-style: none;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+li a {
+  text-decoration: none;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 900;
+}
+
+.ext-more {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 900;
+  color: black;
+}
+
+.ext-more:hover {
+  border-bottom: 8px solid black;
+}
+
+a.extra {
+  color: black;
+}
+
+a.extra:hover {
+  border-bottom: 8px solid black;
+  color: #00bcfc;
+}
+
+a.extratwo {
+  color: #00bcfc;
+}
+
+a.extratwo:hover {
+  border-bottom: 8px solid black;
+}
+
+.ext-more:hover .dropdown-content {
+  display: block;
+}
+
+div.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #f9f9f9;
+  width: fit-content;
+}
+
+.line {
+  height: 1px;
+  background-color: black;
+  width: 84.5rem;
```

Comment:
- [ ] The line underneath the nav links doesn’t reach the end of the page.
- fix this by removing the width